libethereum/State.h

```diff
@@ -123,33 +149,38 @@ struct Change
 		Touch
 	};
 
-	Kind kind;        ///< The kind of the change.
+	Kind kind;		  ///< The kind of the change.
 	Address address;  ///< Changed account address.
-	u256 value;       ///< Change value, e.g. balance, storage and nonce.
-	u256 key;         ///< Storage key. Last because used only in one case.
-	bytes oldCode;    ///< Code overwritten by CREATE, empty except in case of address collision.
+	u256 value;		  ///< Change value, e.g. balance, storage and nonce.
+	u256 key;		  ///< Storage key. Last because used only in one case.
+	bytes oldCode;	///< Code overwritten by CREATE, empty except in case of
+					  ///< address collision.
 
 	/// Helper constructor to make change log update more readable.
-	Change(Kind _kind, Address const& _addr, u256 const& _value = 0):
-			kind(_kind), address(_addr), value(_value)
+	Change(Kind _kind, Address const& _addr, u256 const& _value = 0)
+	  : kind(_kind), address(_addr), value(_value)
 	{
-		assert(_kind != Code); // For this the special constructor needs to be used.
+		assert(_kind !=
+			   Code);  // For this the special constructor needs to be used.
 	}
 
 	/// Helper constructor especially for storage change log.
-	Change(Address const& _addr, u256 const& _key, u256 const& _value):
-			kind(Storage), address(_addr), value(_value), key(_key)
-	{}
+	Change(Address const& _addr, u256 const& _key, u256 const& _value)
+	  : kind(Storage), address(_addr), value(_value), key(_key)
+	{
```

Comment:
I don't know how to keep it like `{}`. Ideas?

Reply:
Working on this now. See [clang reference](https://clang.llvm.org/docs/ClangFormatStyleOptions.html) regarding `SplitEmptyFunction`.

Reply:
I fixed it.